tests/catalyst_slice.py

```diff
@@ -0,0 +1,38 @@
+from catalyst import qjit
```

Comment:
I'm actually not sure its a good idea to have tests here that use catalyst. Could you instead write a unit test that checks the AST transformations work as expected, just using the malt package?